### PR TITLE
UICR-8-Bootstrap-Components

### PR DESCRIPTION
**Pull request by IsaZG**
1. In this pr I added Alerts, Badges, Breadcrumbs, Buttons and Button Groups from Boostrap Documentation: https://getbootstrap.com/docs/4.5/components/alerts/.
2.  It just the scss and html code.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,19 +1,31 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png" />
-    <ui-compone />
-    <ui-comptwo />
-    <HelloWorld msg="Welcome to Your Vue.js + TypeScript App" />
+    <AlertComponent msg="Welcome to Unicoderns Interface Component" />
```
Comment (chrissmejia) [inline src/App.vue:3]: Hi Isa, let's please follow the next pattern for names `<ui-component>` this for example could be `<ui-alert>`

File: src/App.vue [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+    <BadgeComponent />
```
Comment (chrissmejia) [inline src/App.vue:4]: Example `<ui-badge>`

File: package.json [hunk 3]
```diff
@@ -28,11 +28,14 @@
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.2",
     "@vue/test-utils": "^1.0.3",
+    "bootstrap": "^4.5.2",
     "chai": "^4.1.2",
     "eslint": "^6.7.2",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-vue": "^6.2.2",
+    "jquery": "^3.5.1",
```
Comment (chrissmejia) [inline package.json:36]: We will be removing this dependency at some point, JFYI.

File: src/components/BadgeComponent.vue [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="hello">
+    <hr />
+    <h2>Badges</h2>
+    <hr />
+    <h3>Badges size</h3>
+    <h1>Example heading <span class="badge badge-secondary">New</span></h1>
```
Comment (chrissmejia) [inline src/components/BadgeComponent.vue:7]: Please create a `<ui-badge>` that take a prop (param) indicating the kind, ex: secondary, success, etc

File: src/App.vue (hunk 1 above)
Comment (chrissmejia) [reply src/App.vue:3]: I saw you have been grouping by kind, so, for alerts the component can be called `<ui-alerts>` and inside contain multiple `<ui-alert>`

File: src/components/ButtonComponent.vue [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+    <hr />
+    <h3>Button Tags</h3>
+    <a class="btn btn-primary" href="#" role="button">Link</a>
+    <button class="btn btn-primary" type="submit">Button</button>
+    <input class="btn btn-primary" type="button" value="Input" />
+    <input class="btn btn-primary" type="submit" value="Submit" />
+    <input class="btn btn-primary" type="reset" value="Reset" />
+    <hr />
+    <h3>Outline Buttons</h3>
+    <button type="button" class="btn btn-outline-primary">Primary</button>
```
Comment (chrissmejia) [inline src/components/ButtonComponent.vue:23]: Let's do something like `<ui-button type="primary" />`

**Review by chrissmejia — CHANGES_REQUESTED** [review]
Awesome job, just need a couple of changes

**Comment by IsaZG** [comment]
Hi @chrissmejia  I'm going to create another branch and pr and worked with the new commit you created yesterday, then I will send new pr's. Thank you.

**Pull request closed without merging**